src/test/java/com/puppycrawl/tools/checkstyle/TreeWalkerTest.java

```diff
@@ -636,11 +620,14 @@ public void testExternalResourceFiltersWithNoExternalResource() throws Exception
         final DefaultConfiguration checkerConfig = createRootConfig(treeWalkerConfig);
         final File cacheFile = File.createTempFile("junit", null, temporaryFolder);
         checkerConfig.addProperty("cacheFile", cacheFile.getPath());
-
         final String filePath = File.createTempFile("file", ".java", temporaryFolder).getPath();
 
-        final String[] expected = CommonUtil.EMPTY_STRING_ARRAY;
-        verify(checkerConfig, filePath, expected);
+        execute(checkerConfig, filePath);
```

Comment:
What stops is to use verifyWithInlineConfigParser here

Reply:
I dont know how to specify the cacheFile property in the config comment here.
I tried to make xml config like this
```
<module name="Checker">
  <property name="cacheFile" value="target/cachefile"/>
  <module name="com.puppycrawl.tools.checkstyle.TreeWalker">
    <module name="com.puppycrawl.tools.checkstyle.checks.coding.HiddenFieldCheck"/>
  </module>
```
but test coverage decrease and  pitest was falling If you can guide how to specify checkers property (cacheFile) in the config comment  I am all ears

Reply:
yes, make sense.